nil/services/stresser/core/utils.go

```diff
@@ -0,0 +1,16 @@
+package core
+
+import (
+	"crypto/rand"
+
+	"github.com/NilFoundation/nil/nil/common"
+)
+
+func RandomHash() common.Hash {
```

Comment:
It's a copy-paste from SC testaide.

Let's move it into the common and use in all places.

E.g. we have `GenerateRandomAddress` in `types`. So we can add `RandomHash` in `common`.